CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## Version 2.0.1-BETA
```

Comment:
Are we going to have a 2.0.1-beta? I thought we'd just release 2.0.1

Reply:
Ah yeah this needs to be fixed. Thanks for the catch. Just wanted to include the changes. version number didn't thought of it.